ssh/rootfs/etc/ssh/sshd_config

```diff
@@ -23,7 +23,7 @@ HostKey /data/ssh_host_ed25519_key
 # ===================
 Ciphers aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
 MACs -hmac-sha1,hmac-sha1-etm@openssh.com,hmac-sha2-256,hmac-sha2-512,umac-128@openssh.com,umac-64-etm@openssh.com,umac-64@openssh.com
-KexAlgorithms -ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group14-sha256
+KexAlgorithms -curve25519-sha256,curve25519-sha256@libssh.org,diffie-hellman-group-exchange-sha256,diffie-hellman-group14-sha256,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521
```

Comment:
Re removing `diffie-hellman-group-exchange-sha256` -- I'm not sure what their "increase modulus size to 3072 bits or larger" recommendation for this algorithm exactly means, but at least leaving it in and making the `moduli` changes did not result the warning going away.

Reply:
Filed https://github.com/jtesta/ssh-audit/issues/334 asking for clarification